koku/masu/processor/report_parquet_processor_base.py

```diff
@@ -197,13 +198,25 @@ def get_or_create_postgres_partition(self, bill_date, **kwargs):
             if _from == _bill_date:
                 created = _created
             if _created:
-                LOG.info(f"Created a new partition for {record.partition_of_table_name} : {record.table_name}")
+                LOG.info(
+                    log_json(
+                        f"Created a new partition for {record.partition_of_table_name}",
+                        schema=self._schema_name,
+                        table=record.table_name,
```

Comment:
I really try to avoid string formatting unless it is truly necessary. I'm not sure we benefit from formatting this string.
```suggestion
                        msg="created a new partition",
                        schema=self._schema_name,
                        table=record.table_name,
                        partition=record.partition_of_table_name,
```